cdap-docs/integrations/source/index.rst

```diff
@@ -57,9 +57,9 @@ Integrations
 - |mapr|_ Configuring and installing CDAP on **MapR** *(Administration Manual)*
 
 
-.. rubric:: BI Tools: Accessing CDAP Datasets through Business Intelligence Tools
+.. rubric:: Accessing CDAP Datasets through JDBC, ODBC, and Business Intelligence Tools
```

Comment:
We can just say Accessing CDAP Datasets through JDBC and ODBC. BI tools do access them through that.